src/Engines/IChannel.cs

```diff
@@ -20,6 +20,25 @@ public interface IUpdatableDisplay
         void Update(object displayable);
     }
 
+    /// <summary>
+    ///      Provided as a backwards compatability shim for implementations of
```

Comment:
Is this going to stay or temporary?

Reply:
Removing it would be a breaking change, but I think we could safely do so in the not too distant future.